### PR TITLE
t/3: Implemented a keystroke to display the toolbar in a collapsed selection

### LINKED ISSUE #3: Implement a keystroke to display the toolbar in a collapsed selection
It would certainly:
1. Help our users change the block type without making the actual selection.
2. Improve the accessibility of the application.

A follow-up of #1.

### LINKED ISSUE #2344: Implement a keystroke to display the toolbar in a collapsed selection
It would certainly:
1. Help our users change the block type without making the actual selection.
2. Improve the accessibility of the application.

A follow-up of ckeditor/ckeditor5-editor-balloon#1.

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Implemented a keystroke to display the toolbar in a collapsed selection. Closes ckeditor/ckeditor5#2344.

---

Requires https://github.com/ckeditor/ckeditor5-ui/pull/270.

### CODE REVIEW

**Comment by Reinmar** [comment]
Tests?

File: src/balloontoolbareditorui.js [hunk 1]
```diff
@@ -69,7 +69,13 @@ export default class BalloonToolbarEditorUI {
 			origin: editor.editing.view,
 			originFocusTracker: this.focusTracker,
 			originKeystrokeHandler: editor.keystrokes,
-			toolbar: contextualToolbar.toolbarView
+			toolbar: contextualToolbar.toolbarView,
+			beforeFocus: () => {
```
Comment (Reinmar) [inline src/balloontoolbareditorui.js]: `beforeFocus() { ....`

Comment (Reinmar) [reply src/balloontoolbareditorui.js]: BTW, don't you think that `beforeFocus` callback doing `show()` is kinda odd? It's hard to read. I had to look into the code to figure out what's going on here.

Comment (Reinmar) [reply src/balloontoolbareditorui.js]: After a couple more minutes this looks reasonable... :D So I'm ok. But `beforeFocus() { ... }` stays.

**Comment by oleq** [comment]
> Tests?

Well, this is what happens when you start your job one week and finish the next 😅

**Comment by Reinmar** [comment]
❤️  (maybe except that blur at the end, but that's not a priority now)

jul-20-2017 11-14-03

**Comment by oleq** [comment]
AFAIR this blur is a problem we already know about https://github.com/ckeditor/ckeditor5/issues/353.

**Pull request merged**